package.json

```diff
@@ -27,7 +27,7 @@
     "@shapeshiftoss/investor-foxy": "^1.4.1",
     "@shapeshiftoss/investor-yearn": "^1.4.5",
     "@shapeshiftoss/market-service": "^2.2.0",
-    "@shapeshiftoss/swapper": "^2.1.0",
+    "@shapeshiftoss/swapper": "^2.2.1",
```

Comment:
bump this after https://github.com/shapeshift/lib/pull/582 is merged

Reply:
also bump the asset service now that https://github.com/shapeshift/lib/pull/570 is merged and use `assetId` in place of `caip19` everywhere